src/apps/opening-hours/OpeningHoursHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import dayjs from "dayjs";
+import "dayjs/locale/da";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import { DplOpeningHoursListGET200Item as OpeningHoursItem } from "../../core/dpl-cms/model";
+
+dayjs.extend(weekOfYear);
+export const getNextWeek = (date: Date): Date => {
+  return dayjs(date).add(1, "week").toDate();
+};
+
+export const getPreviousWeek = (date: Date): Date => {
+  return dayjs(date).subtract(1, "week").toDate();
+};
+
+export const getWeekDisplay = (date: Date): string => {
+  const week = dayjs(date).week();
+  const year = dayjs(date).year();
+
+  return `Uge ${week}, ${year}`;
```

Comment:
Remember to make this translateable. 

A helper does not make sense in this case.

Reply:
Refactored